README.md

```diff
@@ -23,6 +23,11 @@ An IRC bot of some sort, purpose not clear.
 
 Uses debootstrap. Requires ocpkg.
 
+#### octool_rpi
+For installing opencog on a Raspberry Pi Computer running Raspbian. 
+The readme [here](https://github.com/Dagiopia/opencog_rpi/blob/master/README.md) will be helpful.
+
```

Comment:
It would be great if we could transfer opencog_rpi to opencog organization. That way it is more likely that more people would have eyes on it.

Reply:
okay but know that I got the compiler from https://github.com/raspberrypi/tools and modified a few library files. They have not specified a licence and I don't know if that would be a problem or not.

Reply:
I think it is ok, as long as  we explicitly acknowledge it in the readme, and refer to https://github.com/raspberrypi/tools/issues/58. Give me an admin access to the repo and I would make the transfer.

Reply:
sent u an invite as a collaborator

Reply:
I have cloned and pushed your code to https://github.com/opencog/opencog_rpi as collaborator access doesn't allow me to transfer ownership. You have admin access, so continue your work against that repo. Thanks :smile:

Reply:
:+1: